src/Utilities/CssConcatenator.php

```diff
@@ -150,7 +159,10 @@ private static function getMediaRuleCss(\stdClass $mediaRule): string
      */
     private static function getRuleBlockCss(\stdClass $ruleBlock): string
     {
-        $selectors = \array_keys($ruleBlock->selectorsAsKeys);
-        return \implode(',', $selectors) . '{' . $ruleBlock->declarationsBlock . '}';
+        /** @var string[] $selectors */
```

Comment:
Any reason why not `array<array-key, string>`?

Reply:
No, no particular reason. Changed (and I've also added another type annotation).